syntax = "proto3";

message MuPca9539rStore {
  repeated bool state = 1;
}
